lib/Conversion/PassDetail.h

```diff
@@ -6,8 +6,8 @@
 //
 //===----------------------------------------------------------------------===//
 
-#ifndef DIALECT_EMITC_CONVERSION_PASSDETAIL_H
-#define DIALECT_EMITC_CONVERSION_PASSDETAIL_H
+#ifndef CONVERSION_PASSDETAIL_H
+#define CONVERSION_PASSDETAIL_H
```

Comment:
```suggestion
#ifndef EMITC_CONVERSION_PASSDETAIL_H
#define EMITC_CONVERSION_PASSDETAIL_H
```
Good fetch! However, I assume the suggested include guard might be used in other projects.

Maybe we should make this *more unique*, like Torch-MLIR does, see [here](https://github.com/llvm/torch-mlir/blob/c8e062fb4e8de4620b6ae5e145a9da9a7cca8e5c/lib/Conversion/PassDetail.h#L10-L11). There, `TORCHMLIR_` is used as prefix which closely resembles the project name **and** the name of the include directory (`torch-mlir`). Suggesting `EMITC` to match with our include dir.